packages/portfolio-contract/src/constants.js

```diff
@@ -11,3 +11,15 @@ export const YieldProtocol = /** @type {const} */ ({
   USDN: 'USDN',
 });
 harden(YieldProtocol);
+
+// TODO: pass via terms?
+/**
+ * @enum {(typeof PortfolioChain)[keyof typeof PortfolioChain]}
+ */
+export const PortfolioChain = /** @type {const} */ ({
```

Comment:
I'm not clear on what this type is for. As far as I can tell, the code treats all `CaipChainId`s the same. Help me understand?